passkey-api/introduction.mdx

```diff
@@ -0,0 +1,229 @@
+    	const creationOptions = await fetch(baseUrl + "/registration/initialize", {
+    		method: "POST",
+    		headers,
+    		body: JSON.stringify({
+    			user_id: user.id.toString(), // Must be a string!
+    			username: user.username,
+    		}),
+    	}).then((res) => res.json());
+
+    	// creationOptions is an object that can directly be passed to create()
+    	// (the function that opens the "create passkey" dialog)
+    	// in the frontend.
+    	res.json(creationOptions);
+    });
+
+    app.post("/passkey/finalize-registration", async (req, res) => {
+    	const data = await fetch(baseUrl + "/registration/finalize", {
+    		method: "POST",
+    		headers,
+    		body: JSON.stringify(req.body), // Forward newly created credential
+    	}).then((res) => res.json());
+
+    	// The response from the Passkey API contains a JWT (`data.token`).
+    	// What you do with this JWT is up to you.
+    	//
+    	//
+    	// The JWT contains 4 claims:
+    	// "sub": the user_id we used for "/registration/initialize".
+    	//        you can use this to issue a session for the user, for example
+    	//
+    	// "cred": the credential_id (ID of the passkey the user chose)
+    	//
+    	// "aud": an array with a single string.
+    	//        the string is the ID of the relying party (your app).
+    	//        e.g. ["example.com"]
+    	//
+    	// "iat": the expiration date (very short)
+    	//
+    	//
+    	// Here, we don't need to do anything with it, since the user already
+    	// is logged in. In the login endpoints, later in this guide, we'll
+    	// use the data contained in the JWT to create a session for our user.
+
+    	res.redirect("/success");
+    });
+    ```
+
+    **Frontend:**
+
+    ```jsx
+    import { create, get } from "@github/webauthn-json";
+
+    async function registerPasskey() {
+    	// Let's send a request to our backend to start the registration process.
+    	// The response JSON can directly be passed to create(...) below.
+    	const creationOptions = await fetch("/passkey/start-registration", {
+    		method: "POST",
+    	}).then((res) => res.json());
+
+    	// Open "create passkey" dialog
+    	const credential = await create(creationOptions);
+
+    	// User successfully created a passkey on their device.
+    	//
+    	// The resulting `credential` object needs to be sent back to the
+    	// Passkey API as-is, through our backend:
+    	//
+    	// frontend → backend → passkey API
+    	return fetch("/passkey/finalize-registration", {
+    		method: "POST",
+    		headers: { "Content-Type": "application/json" },
+    		body: JSON.stringify(credential),
+    	});
+    }
+    ```
+
+    Here's what the whole flow looks like:
+
+    <Frame>
+    	<img src="/images/passkey-api/register-flow.png" />
+    </Frame>
+
+    As you can see, there's two steps here ("start" and "finalize"), which pass through the frontend, backend, and Passkey API.
+
+    The process looks very, very similar for logging in:
```

Comment:
I didn't get this line. Do you plan to add something after this line?

Reply:
No, it's just supposed to be a lead-in for the _Login_ section